### PR TITLE
Barcode manual override

### PR DESCRIPTION
**Pull request by tannermpeterson**

### CODE REVIEW

File: tests/server/test_server_routes_populate_queues.py [hunk 1]
```diff
@@ -804,21 +806,65 @@ def test_start_recording_command__populates_queue__with_correctly_parsed_set_of_
...
     communication = comm_queue.get(timeout=QUEUE_CHECK_TIMEOUT_SECONDS)
     assert communication["command"] == "start_recording"
     assert set(communication["active_well_indices"]) == set([0, 8, 5])
 
 
+@pytest.mark.parametrize(
+    "scanned_barcode,user_entered_barcode,expected_result,test_description",
+    [
+        (
+            "MA200440001",
```
Comment (eli88fine) [inline tests/server/test_server_routes_populate_queues.py:826]: do you think it's relevant to test an empty string `""`?  I don't recall what ok_comm passes back to processMonitor if the barcode fails to scan

**Review by eli88fine — APPROVED** [review]
looks good. please take a look at minor comments/questions and determine if they are relevant to address before merging or not

File: tests/server/test_server_routes_populate_queues.py (hunk 1 above)
Comment (tannermpeterson) [reply tests/server/test_server_routes_populate_queues.py:826]: It does pass an empty string when the scanner fails. I can add that test in the next PR fairly quickly

File: tests/server/test_server_routes_populate_queues.py [hunk 2]
```diff
@@ -804,21 +806,65 @@ def test_start_recording_command__populates_queue__with_correctly_parsed_set_of_
         "metadata_to_copy_onto_main_file_attributes"
     ][PLATE_BARCODE_UUID]
     response = test_client.get(
-        f"/start_recording?barcode={expected_barcode}&active_well_indices=0,5,8&is_hardware_test_recording=False"
+        f"/start_recording?barcode={expected_barcode}&active_well_indices=0,5,8"
```
[the remark this answers is not in the document]
Comment (tannermpeterson) [reply tests/server/test_server_routes_populate_queues.py:809]: False is the default value here and we already have other tests that confirm that test that functionality

[the remark this answers is not in the document]
Comment (tannermpeterson) [reply tests/server/test_server_routes_populate_queues.py:809]: Oh nvm, it's the default in `shared_values_dict`. I'll add it back but I think that regardless it shouldn't affect what we're testing here

**Pull request merged**